tensorboard/backend/application_test.py

```diff
@@ -506,20 +509,41 @@ def setUp(self):
               FakePlugin,
               plugin_name='bar',
               is_active_value=True,
-              routes_mapping={'/bar_route': self._bar_handler},
+              routes_mapping={
+                '/bar_route': self._bar_handler,
+                '/wildcard/*': self._wildcard_handler
+                },
```

Comment:
nit: Please indent the closing delimiter of containers to the same level as
the line on which the container was opened:

```python
foo(
    routes_mapping={
        # ...
    },
    some_list=[
        # ...
    ],
)
```

Reply:
done